crates/sui-framework/tests/natives_calibration_tests.move

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
I'd be worried about doing all this in the source language since you have less guarantees about what instructions you are actually emitting. Have you given any thought about that?  I know we had discussed briefly about using the mvir to invoke these functions